README.md

```diff
@@ -48,21 +48,54 @@ We recommend cid-cmd tool via [AWS CloudShell](https://console.aws.amazon.com/cl
     ```bash
     cid-cmd deploy
     ```
-#### Optional Commands
+### Demo
 
-6. Update existing Dashboards
-    ```bash
-    cid-cmd update
-    ```
+   [![asciicast](https://asciinema.org/a/467770.svg)](https://asciinema.org/a/467770)
 
-7. See available commands and command line options
-    ```
-    cid-cmd --help
-    ```
+## Other Commands
 
-### Usage Demo
+#### Update existing Dashboards
+Update only Dashboard
+```bash
+cid-cmd update
+```
+Update dashboard and all dependenies (Datasets and Athena View). WARNING: this will override any customization of SQL files and Datasets.
+```bash
+cid-cmd update --force --recursive 
+```
+#### Show Sashboard status 
+```bash
+cid-cmd status
+```
+
+####  Share QuickSight resources
+```bash
+cid-cmd share
+```
+
+#### Delete Dashboard and all dependencies unused by other
```

Comment:
what does unused by other mean? Should we just say "delete all dashboards and all dependencies" ?

Reply:
This actually delete one dashboard and deps, but only if this dep is not used by other dashboards; so we can delete kpi and keep cudos even when they use the same view